r/R/dplyr-functions.R

```diff
@@ -526,14 +526,6 @@ nse_funcs$second <- function(x) {
   Expression$create("add", Expression$create("second", x), Expression$create("subsecond", x))
 }
 
-# After ARROW-13054 is completed, we can refactor this for simplicity
```

Comment:
I don't know if you want to do it, but the Python side probably needs to be updated in order to expose the options class.

Reply:
It's ready in [ARROW-12980](https://github.com/apache/arrow/pull/10457/files#diff-2010b86c11cb90f0ce6ef00648df2e388d11cda5d3fc74e4293dd862e228daec). Would it make sense to copy-paste it here too?

Reply:
That assumes you're using the same options class for all these kernels, which seems a bit weird to me (why would the `year` kernel take unrelated options?).

Reply:
Ah, I misunderstood the original question. I'll add the options to Python.

Reply:
Done.